Cargo.toml

```diff
@@ -1,34 +1,34 @@
 [workspace]
 members = [
-  "crates/bench",
-  "crates/bindings-sys",
-  "crates/bindings",
-  "crates/bindings-macro",
-  "crates/cli",
-  "crates/client-api",
-  "crates/client-api-messages",
-  "crates/core",
-  "crates/data-structures",
-  "crates/lib",
-  "crates/metrics",
-  "crates/primitives",
-  "crates/sats",
-  "crates/sdk",
-  "crates/sqltest",
-  "crates/standalone",
-  "crates/table",
-  "crates/testing",
-  "crates/vm",
-  "modules/benchmarks",
-  "modules/rust-wasm-test",
-  "modules/quickstart-chat",
-  "modules/sdk-test",
-  "modules/sdk-test-connect-disconnect",
-  "modules/spacetimedb-quickstart",
-  "crates/sdk/tests/test-client",
-  "crates/sdk/tests/test-counter",
-  "crates/sdk/tests/connect_disconnect_client",
-  "tools/upgrade-version",
+    "crates/bench",
```

Comment:
Why the formatting change?

Reply:
I'd rather not pollute the git history

Reply:
Done. Look like JetBrains add a formatter for toml.